test/unit/org/apache/cassandra/io/sstable/CQLSSTableWriterTest.java

```diff
@@ -1116,6 +1118,122 @@ public void testFrozenSetTypeCustomOrdered() throws Exception
         assertEquals(0, filtered.size());
     }
 
+    @Test
+    public void testWriteWithTimestamps() throws Exception
+    {
+        long now = currentTimeMillis();
+        long then = now - 1000;
+        final String schema = "CREATE TABLE " + qualifiedTable + " ("
+                              + "  k int,"
+                              + "  c1 int,"
+                              + "  c2 int,"
+                              + "  v text,"
+                              + "  PRIMARY KEY (k)"
```

Comment:
this is confusing, `c1` and `c2` normally means "clustering column" when you read the code, but this does not include it; can you rename these to v1, v2, and v3?  or change the test to make these clustering columns?